db/migrate/20190517152315_remove_null_false_constraint_from_invitation_token_on_memberships.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveNullFalseConstraintFromInvitationTokenOnMemberships < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null :memberships, :invitation_token, true
+    change_column_null :memberships, :invited_by_id, true
+    change_column_null :memberships, :confirmed_at, true
```

Comment:
This is a bit ugly. Rails was trying to create the membership object first when creating it through the user, this was causing db validation errors on the `NOT NULL` constraint.

Had a go at fixing it in code like so but failed: https://thoughtbot.com/blog/accepts-nested-attributes-for-with-has-many-through

So in the end just removed the validations.